src/components/touchables/GenericTouchable.tsx

```diff
@@ -2,12 +2,13 @@ import * as React from 'react';
 import { Component } from 'react';
 import {
   Animated,
-  Platform,
   StyleProp,
   ViewStyle,
   TouchableWithoutFeedbackProps,
   Insets,
-} from 'react-native';
+} from '../../ReactNativeImports';
+
+import { Platform } from '../../ReactCompat';
```

Comment:
Actually can we merge both `ReactNativeImports` and  `ReactCompat` files into single one? Is there any reason why we keep it separately?

Reply:
Yes, we can 😄

Done in [60268dc](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/60268dcef32d4df2a9d11b576a2f16a842baa7d2).